docbook/Admin_Guide/en-US/config/auth.xml

```diff
@@ -91,10 +91,9 @@
 
 					<itemizedlist>
 						<listitem>
-							<para>Protocol can be either
-								<filename>ldap</filename> or
-								<filename>ldaps</filename> (for SSL/TLS encryption).
-								If omitted, then an unencrypted connection will be established on port 389.
+							<para>Protocol must be either:
+								<filename>ldap</filename> (unencrypted or opportunistic StartTLS) or
+								<filename>ldaps</filename> (for TLS encryption).
```

Comment:
I don't think _filename_ tag is appropriate here - suggest `emphasis` instead. 
Suggest formatting as unordered list, as mentioned in my ealier comment on config_defaults_inc.php

Reply:
Fixed.